Add tenting wedge files

violet-fish opened the pull request.

violet-fish committed:
Add tenting wedge files

violet-fish committed:
Remove worse version of tenting wedge

- Remove inferior wedge from repo
- Rename the good wedge to remove now redundant version number

ploopyco commented:
"ploopy_mini_wedge" seems a bit redundant. Could you change the file names to just "Wedge"?

violet-fish committed:
Rename tenting files to be a bit less redundant

- Specifying the wedge is for the ploopy mini in the mini repo is
not necessary
